Formula/appscale-tools.rb

```diff
@@ -14,8 +14,11 @@ class AppscaleTools < Formula
     sha256 cellar: :any, high_sierra: "70e89498336894ae025118e51e418528d8d73da9b1e2786559b6bcbe6055f55b"
   end
 
+  deprecate! date: "2020-04-20", because: "depends on Python 2"
```

Comment:
Any reason for this date? Probably better to just make it today's date so we know when we actually deprecated it.

Reply:
The reason I chose was because it was the day after the release of Python 2.7.18.  I happened to see this used for `libgraphqlparser`, but it seems that this was merged even though there a desire to use the present date instead.  So I will just use today's date instead (for me in the US).